R/avg.R

```diff
@@ -74,7 +78,8 @@ avg_event <- function(id, group, par = NULL) {
           group = unique(X$group),
           par = unique(X$par),
           avg = mean(X$n, na.rm = TRUE),
-          se = sd(X$n, na.rm = TRUE) / sqrt(nrow(X))
+          se = sd(X$n, na.rm = TRUE) / sqrt(nrow(X)),
+          count = nrow(X)
```

Comment:
Same comment on Line 59. If this should be counts of events, the following is suggested:
```suggestion
          count = sum(X$n)
```